pkg/utils/strings.go

```diff
@@ -0,0 +1,10 @@
+package utils
+
```

Comment:
In general in go we do not create utility packages like this.  This is a trivial method and would inline privately in a package.  Utility packages cause dependency creep and attract unrelated use cases, which cause coupling.

Please move this to a private method at the package call site.

Reply:
Thank you for the comment. It makes sense, I will remove this in some further PR.